UResearcher/uresearcher_app/controllers/modules/latent_knowledge_analysis.py

```diff
@@ -52,35 +53,71 @@ def get_wordvecs(articles):
 			for curr_sentence in article_sentences:
 			
 				preproc = sentence_parsing(curr_sentence)
-			
+				"""
+				for s in article_sentences:
+					tokenized_ = nltk.word_tokenize(curr_sentence)
+					tuple_list = nltk.pos_tag(tokenized_)
+				"""
 				#sentences.append(tokenize.word_tokenize(preproc))
 				#print(tokenize.word_tokenize(curr_sentence))
 				
 				
 				sentences.append(preproc)
 				
 
-	
-	model = Word2Vec(sentences, min_count=3, workers=cores-1)
+	print("start word2vec")
+
+	#https://radimrehurek.com/gensim/models/word2vec.html
+	#model = Word2Vec(sentences, min_count=3,vector_size=1000,max_vocab_size=2000,max_final_vocab=None,sorted_vocab=1,  workers=cores-1)
+	model = Word2Vec(sentences, min_count=3, vector_size=100,sorted_vocab=1,
+					 workers=cores - 1)
 	#model.build_vocab(sentences, progress_per=10000)
+	# we can edit the dictionary here to remove verb or some unnecsary word
+
+	#
 
 
 	return model.wv
 
-def get_2d_projection(articles):
+def get_vocablist(articles,filter):
+	i=0
+	vectors = get_wordvecs(articles)
+	vocab=[]
+	for word in vectors.index_to_key:
+		if i > 2000:
```

Comment:
Please dont hard code the values. We want to avoid hardcoding values in the future.